rplugin/python3/denite/source/file_rec.py

```diff
@@ -6,32 +6,30 @@
 
 from .base import Base
 from denite.process import Process
-from os import path, pardir
+from os import path
 from os.path import relpath, isabs, isdir, join
 from denite.util import parse_command, abspath
 import argparse
 import os
+import vim
```

Comment:
This works for me when I run it from inside vim but it fails the CI checks.
Can you advice here how to fix this?

@Shougo

Reply:
Please don't use this.
Please use `self.vim` instead.